config/gni/devtools_grd_files.gni

```diff
@@ -540,9 +540,7 @@ grd_files_release_sources = [
   "front_end/panels/recorder/recorder.js",
   "front_end/panels/recorder/util/util.js",
   "front_end/panels/rn_welcome/rn_welcome-meta.js",
-  "front_end/panels/rn_welcome/RNWelcome.js",
```

Comment:
This isn't a correct change - the files are still included in the build, of which this config file is a simple manifest. See https://github.com/motiz88/rn-chrome-devtools-frontend/actions/runs/6263653657/job/17008522943?pr=11#step:5:1999.

I don't think we should be removing the files though - disabling the UI should be enough.

Reply:
Yeah this did the opposite on macOS locally, prompting the edit (I assumed it was doing some sort of tree shaking based on experiments). Either way yeah it's better to remove all the files, and I agree with decoupling from `REACT_NATIVE_SPECIFIC_UI`.